db/migrate/20150312144232_migrate_websockets_setting.rb

```diff
@@ -0,0 +1,16 @@
+class MigrateWebsocketsSetting < ActiveRecord::Migration
+  def up
+    old_value = Setting.find_by_name("websockets_encrypt")
+    new_value = (old_value == true || old_value == "true" || old_value == "auto") ? true : false
+    Setting.find_by_name("websockets_encrypt").update_attributes(
```

Comment:
Tests fail here when migrating a blank database, as the setting may not exist yet.

<pre>
00:03:20.410 undefined method `update_attributes' for nil:NilClass/var/lib/workspace/workspace/test_develop_pr_core/database/sqlite3/ruby/1.9.3/db/migrate/20150312144232_migrate_websockets_setting.rb:5:in `up'
</pre>